INSTALL.md

```diff
@@ -1028,7 +1027,7 @@ Run from MSYS console in the source directory:
 
 We're going to use precompiled binaries. Use the link below for download:
 
-> http://wwwmaster.postgresql.org/download/mirrors-ftp?file=%2Fbinary%2Fv8.2.4%2Fwin32%2Fpostgresql-8.2.4-1-binaries-no-installer.zip
+http://wwwmaster.postgresql.org/download/mirrors-ftp?file=%2Fbinary%2Fv8.2.4%2Fwin32%2Fpostgresql-8.2.4-1-binaries-no-installer.zip
```

Comment:
This link is broken. And postgres 8.2 is very outdated.
Not sure how we can make this section more future proof and less "maintenance heavy".

Reply:
Do we actually need this whole section [4.3 Creation of MSYS environment for compilation of QGIS](https://github.com/qgis/QGIS/blob/master/INSTALL.md#43-creation-of-msys-environment-for-compilation-of-qgis)? We provide an archive of msys at  [4.2.1. MSYS](https://github.com/qgis/QGIS/blob/master/INSTALL.md#421-msys) and knowing that we claim [few lines above (4.2)](https://github.com/qgis/QGIS/blob/master/INSTALL.md#42-building-using-mingw) that building the msys/minGW way is not the preferred road.
IOW do we need to provide ways to do everything from scratch, at the risk of having long and outdated instructions, while we already provide the necessary bricks?

Reply:
I am very open to removing outdated sections (which have been proven that they are not maintained in years)

Reply:
The msys instructions are unmaintained - and the link to the msys.zip is broken too.

Reply:
@jef-n Any pointer to the new link? I looked into https://download.osgeo.org/ with no success; Or do we no longer provide the msys.zip? In which case the only instruction we should keep would be the one suggesting Visual Studio?